queries/tlaplus/highlights.scm

```diff
@@ -0,0 +1,140 @@
+; highlights.scm
+
+; Keywords
+[
+  "ACTION"
+  "ASSUME"
+  "ASSUMPTION"
+  "AXIOM"
+  "BY"
+  "CASE"
+  "CHOOSE"
+  "CONSTANT"
+  "CONSTANTS"
+  "COROLLARY"
+  "DEF"
+  "DEFINE"
+  "DEFS"
+  "DOMAIN"
+  "ELSE"
+  "ENABLED"
+  "EXCEPT"
+  "EXTENDS"
+  "HAVE"
+  "HIDE"
+  "IF"
+  "IN"
+  "INSTANCE"
+  "LAMBDA"
+  "LEMMA"
+  "LET"
+  "LOCAL"
+  "MODULE"
+  "NEW"
+  "OBVIOUS"
+  "OMITTED"
+  "ONLY"
+  "OTHER"
+  "PICK"
+  "PROOF"
+  "PROPOSITION"
+  "PROVE"
+  "QED"
+  "RECURSIVE"
+  "SF_"
+  "STATE"
+  "SUBSET"
+  "SUFFICES"
+  "TAKE"
+  "TEMPORAL"
+  "THEN"
+  "THEOREM"
+  "UNCHANGED"
+  "UNION"
+  "USE"
+  "VARIABLE"
+  "VARIABLES"
+  "WF_"
+  "WITH"
+  "WITNESS"
+] @keyword
+[
+  (forall)
+  (exists)
+  (temporal_forall)
+  (temporal_exists)
+  (set_in)
+  (def_eq)
+  (gets)
+  (maps_to)
+  (all_map_to)
+  (case_box)
+  (case_arrow)
+  (label_as)
+] @keyword.function
```

Comment:
Until now we only have keyword.function for keywords the introduce a function definition. Are those keywords  connected to similar concept? They seem to be able to introduce new clauses

Reply:
Hmmm, a couple of them roughly meet that definition but not all; I will experiment then make some changes.